protons/driver.py

```diff
@@ -974,7 +1003,7 @@ def _attempt_state_change(self, context, reject_on_nan=False):
                     self.nrejected += 1
                 self.pot_energies.append(pot1)
                 self.kin_energies.append(kin1)
-                # Restore sams_sampler states.
+                # Restore titration states.
                 for titration_group_index in titration_group_indices:
                     self._set_titration_state(titration_group_index, initial_titration_states[titration_group_index], context)
                 # If using NCMC, restore coordinates and flip velocities.
```

Comment:
The `_ncmc_ghmc` and `_attempt_state_change` functions are unaffected, which is good for us.